tutorials/releases.md

```diff
@@ -0,0 +1,311 @@
+![CRI-O logo](../logo/crio-logo.svg)
+
+Welcome to the releases page! Enjoy CRI-O!
+
+- [1.18 (latest stable)](#118--latest-stable-)
+  * [1.18.0](#1180)
+    + [CentOS 8](#centos-8)
+  * [CentOS 8 Stream](#centos-8-stream)
+    + [Debian Unstable](#debian-unstable)
+    + [Debian Testing](#debian-testing)
+    + [Fedora 31 or later](#fedora-31-or-later)
+    + [Ubuntu 20.04](#ubuntu-2004)
+- [1.17](#117)
+  * [1.17.4](#1174)
+    + [CentOS 7](#centos-7)
+    + [CentOS 8](#centos-8-1)
+    + [CentOS 8 Stream](#centos-8-stream-1)
+    + [Debian Unstable](#debian-unstable-1)
+    + [Debian Testing](#debian-testing-1)
+    + [Fedora](#fedora)
+    + [Ubuntu 18.04](#ubuntu-1804)
+    + [Ubuntu 19.04](#ubuntu-1904)
+    + [Ubuntu 19.10](#ubuntu-1910)
+    + [Ubuntu 20.04](#ubuntu-2004-1)
+- [1.16](#116)
+  * [1.16.6](#1166)
+    + [CentOS 7](#centos-7-1)
+    + [CentOS 8](#centos-8-2)
+    + [CentOS 8 Stream](#centos-8-stream-2)
+    + [Debian Unstable](#debian-unstable-2)
+    + [Debian Testing](#debian-testing-2)
+    + [Fedora](#fedora-1)
+    + [Ubuntu 18.04](#ubuntu-1804-1)
+    + [Ubuntu 19.04](#ubuntu-1904-1)
+    + [Ubuntu 19.10](#ubuntu-1910-1)
+    + [Ubuntu 20.04](#ubuntu-2004-2)
+- [1.15 (deprecated)](#115--deprecated-)
+  * [1.15.4](#1154)
+    + [CentOS 7](#centos-7-2)
+    + [Debian Unstable](#debian-unstable-3)
+    + [Debian Testing](#debian-testing-3)
+    + [Fedora](#fedora-2)
+
+
+# 1.18 (latest stable)
+**Notes**:
+  - CRI-O 1.18.0 or higher version **requires golang 1.13 or higher**
+
+## 1.18.0
+### CentOS 8
+For CentOS 8 run the following as root:
+```
+cd /etc/yum.repos.d/
+wget https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:1.18:1.18.0/CentOS_8/devel:kubic:libcontainers:stable:cri-o:1.18:1.18.0.repo
```

Comment:
I guess this will get updated for every release, yes? Are you specifying the generic major.minor repo as well, so people who always want to stay on the latest patch release don't have to add a new repo everytime?

Reply:
Also, need to specify the main project repo as well, to fetch dependencies like conmon (unless you assume that people have that repository added already)